.github/dependabot.yml

```diff
@@ -14,5 +14,5 @@ updates:
         # Included when device-sdk-go is updated
       - dependency-name: "github.com/edgexfoundry/go-mod-secrets/v4"
         # Included when device-sdk-go is updated
-      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/v4"
+      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
```

Comment:
please update all the depencencies

Reply:
is there any samples? which repo can I refer?

Reply:
```
    ignore:
      - dependency-name: "github.com/edgexfoundry/go-mod-core-contracts/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-messaging/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-registry/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-secrets/*"
        # Included when device-sdk-go is updated
      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
        # Included when device-sdk-go is updated    
```

Reply:
got it.